src/github.com/getlantern/elevate/command.go

```
@@ -0,0 +1,37 @@
+// package elevate provides support for executing commands with elevated
+// privileges.
+package elevate
+
+import (
+	"os/exec"
+)
+
+// Command is like exec.Command, except that it runs the given command with
+// elevated privileges.
+func Command(name string, args ...string) *exec.Cmd {
+	cmd, err := buildCommand("", "", name, args...)
+	if err != nil {
+		panic(err)
```

What's your consideration of using panic rather than return the error?